docs/guide/index.md

```diff
@@ -0,0 +1,70 @@
+# Guide
+The Official Guide For Pycord
+
+# Quick Tip
+Pycord comes with automatic generation of files using your command prompt.
+
+Example some commands you can use:
+
+``python -m py-cord newbot (args)``
+
+#### Args:
+
+- name
+  - Your Project Name
+- --sharded
+   - If your bot should be sharded or not
+- --prefix
+   - Your Bot Prefix
+
+``python -m py-cord newcog (args)``
+
+#### Args:
+
+- name
+   - The name of your Cog
+
+- directory 
+   - Cogs Directory
+
+- --full
+   - Get Full Features Of Cogs
+
+- --class-name
+   - Name of Your Cog Class 
+
+## Before you begin...
+Pycord has a lot of features which would be too advanced for a person just starting out with python,
+We would suggest you get the basic knowledge of Python before starting there is a lot of tutorials to follow and we would suggest you start off with small projects then get bigger as you progress.
+
+### How much python do i need to know?
+
+- The difference between instances and class attributes.
+    - e.g. `guild.name` vs `discord.Guild.name` or any variation of these.
+- How to use data structures in the language.
+    - `dict`/`tuple`/`list`/`str`/`...`
+- How to solve `NameError` or `SyntaxError` exceptions.
+- How to read and understand tracebacks.
+
```

Comment:
I think we should also add
"Basic understanding of asyncio"

Reply:
you don't really need to know asyncio to use pycord

Reply:
I suggested this because most of the time I see users stuck due to their lack of understanding in how async stuff works in python

And also I meant "Basic understanding of how async things work in python" not asyncio :oops: